twake/frontend/src/app/components/OnBoarding/popups/VerifyEmail.tsx

```diff
@@ -1,14 +1,28 @@
-import { Button, Col, Row, Typography } from 'antd';
+import { Button, Row, Typography } from 'antd';
 import React, { useState } from 'react';
 import Emojione from '../../Emojione/Emojione';
 import ObjectModal from '../../ObjectModal/ObjectModal';
 import Languages from 'services/languages/languages.js';
+import ModalManager from 'app/components/Modal/ModalManager';
+import AddMailsInWorkspace from './AddMailsInWorkspace';
+import Api from 'app/services/Api';
 
-type PropsType = {};
+type PropsType = {
+  email: string;
+};
+
+const VerifyMail = ({ email }: PropsType) => {
+  const openAddMembers = () =>
+    ModalManager.open(<AddMailsInWorkspace />, {
+      position: 'center',
+      size: { width: '600px' },
+    });
+
+  const onClickButton = () => {
+    const route = 'users/console/api/verify_mail';
 
-const VerifyMail = ({}: PropsType) => {
-  const [email, setEmail] = useState<string>('dreamteam@linagora.com');
-  const onClickButton = () => console.log('clicked');
+    return Api.post(route, {});
+  };
```

Comment:
This is quite useless to declare a variable for route to use it once. Also, instead of having to do this messy stuff, a dedicated service like `ConsoleService` with a `verifyEmail` function will be better, even if it is 2 lines long

Reply:
done 👍🏻